### ISSUE TITLE
Parameter estimates do not converge

### ISSUE THREAD

**Issue by simsurace** [Theoretical-Neuroscience-Group/BinomialSynapses.jl#17]
There is currently no automated test for this.

**Comment by simsurace** [comment]
This is probably due to the `likelihood_resample` function using the indices in the wrong way, as suggested by @camillegontier. We should have something like
```
    for j in 1:M_out
        ns[j,:].=ns[j,idx[j,:]]
        ks[j,:].=ks[j,idx[j,:]]
    end
```

**Comment by simsurace** [comment]
Should be fixed by [`bd2ddde`](https://github.com/Theoretical-Neuroscience-Group/BinomialSynapses.jl/commit/bd2ddde4e76f283a210bfcfa820c3b5ef82b5ebd).

**Comment by simsurace** [comment]
@camillegontier reported that this is fixed
